client/components/seo/seo-site-search-preview.jsx

```diff
@@ -0,0 +1,39 @@
+import React, { PropTypes } from 'react';
+import PureRenderMixin from 'react-pure-render/mixin';
+
+export const SeoSiteSearchPreview = React.createClass( {
+	mixins: [ PureRenderMixin ],
+
+	render() {
+		const {
+			snippet,
+			title,
+			url
+		} = this.props;
+
+		return (
+			<div className="seo-search-result-preview">
+				<h2 className="seo-search-result-preview__header">{ this.translate( 'Google Preview' ) }</h2>
```

Comment:
I mean the actual heading string: 'Google Preview'. Change to 'Search Preview'? Sorry to confuse ;)